src/Model/Item.php

```diff
@@ -1988,6 +1991,35 @@ public static function insert($item, $notify = false, $dontcache = false)
 		return $current_post;
 	}
 
+	/**
+	 * Convert items to forum posts
+	 *
+	 * @param array $item
+	 * @return void
+	 */
+	private static function transformToForumPost(array $item)
+	{
+		if ($item["verb"] != Activity::ANNOUNCE) {
+			return;
+		}
```

Comment:
Please move this condition and wrap this method call on line 1801 with it.

The reasoning is that if the method is called, according to the name we will transform it to a forum post. Whether we should do it or not is irrelevant to this methods.

Reply:
Not every announce should be converted, only the ones from forum accounts. I think that this check and the check in lines 2006-2009 belong together. I think it decreases readability when these two checks are split.